```
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Checking for module 'uuid'
--   Found uuid, version 2.27.0
-- Output libraries to $SRC_DIR/dist
CMake Error at /tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/surelog_1574224338484/_build_env/share/cmake-3.15/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find SWIG (missing: SWIG_EXECUTABLE SWIG_DIR)
Call Stack (most recent call first):
  /tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/surelog_1574224338484/_build_env/share/cmake-3.15/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  /tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/surelog_1574224338484/_build_env/share/cmake-3.15/Modules/FindSWIG.cmake:64 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:8 (FIND_PACKAGE)
-- Configuring incomplete, errors occurred!
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -38,6 +38,7 @@ env:
   - PACKAGE=tree-sitter-verilog
   - PACKAGE=slang               CONDA_CHANNELS="conda-forge"    USE_SYSTEM_GCC_VERSION="8"
   - PACKAGE=moore
+  - PACKAGE=surelog             CONDA_CHANNELS="conda-forge"
   - PACKAGE=sv-parser
   - PACKAGE=verible                                             USE_SYSTEM_GCC_VERSION="8"
   # protocol analyzers

--- a/surelog/build.sh
+++ b/surelog/build.sh
@@ -1,0 +1,16 @@
+#! /bin/bash
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+  CPU_COUNT=2
+fi
+
+mkdir build && cd build
+
+cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_VERBOSE_MAKEFILE=1
+
+make -j$CPU_COUNT install
+
+

--- a/surelog/meta.yaml
+++ b/surelog/meta.yaml
@@ -1,0 +1,35 @@
+package:
+  name: surelog
+  version: {{ GIT_FULL_HASH }}
+
+source:
+  git_url: https://github.com/alainmarcel/Surelog
+  git_rev: master
+
+build:
+  number: {{ environ.get('DATE_NUM') }}
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - TRAVIS
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+  host:
+    - python {{ python }}
+    - gperftools
+    - tk
+    - libunwind
+    
+  run:
+    - python
+
+about:
+  home: https://github.com/alainmarcel/Surelog
+  license: Apache
+  license_file: LICENSE
+  summary: 'Parser/Compiler for SystemVerilog'

--- a/surelog/meta.yaml
+++ b/surelog/meta.yaml
@@ -1,6 +1,7 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('-v','.') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
 package:
   name: surelog
-  version: {{ GIT_FULL_HASH }}
+  version: {{ version }}
 
 source:
   git_url: https://github.com/alainmarcel/Surelog
@@ -27,7 +28,7 @@ requirements:
     - libunwind
     - libuuid
     - swig
-    
+
   run:
     - python
 

--- a/surelog/meta.yaml
+++ b/surelog/meta.yaml
@@ -18,12 +18,15 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
+    - pkg-config
 
   host:
     - python {{ python }}
     - gperftools
     - tk
     - libunwind
+    - libuuid
+    - swig
     
   run:
     - python

